mmpy_bot/plugins/example.py

```diff
@@ -110,7 +119,13 @@ async def ping_reply(self, message: Message):
         """Pong."""
         self.driver.reply_to(message, "pong")
 
-    @listen_to("^reply at (.*)$", re.IGNORECASE, needs_mention=True)
+    @listen_to(
+        "^reply at (.*)$",
+        re.IGNORECASE,
+        needs_mention=True,
+        category="schedule",
+        syntax="reply at TIMESTAMP",
```

Comment:
I'd prefer storing this kind of information in the docstring rather than an explicit `syntax` argument here; it feels unnatural.